pkg/services/ngalert/notifier/alert_reception.go

```diff
@@ -13,6 +13,11 @@ import (
 	"github.com/prometheus/common/model"
 )
 
+// DefaultResolveTimeout is the default timeout use for resolving the alert
+// if the end time for alert is not specified.
+// TODO: should this be configurable?
+const DefaultResolveTimeout = 30 * time.Minute
```

Comment:
The alertmanager default is 5 minutes, should we do the same? I'd not worry about making it configurable for now. Unless there was a similarly thing in Grafana, perhaps @papagian would know?

Reply:
I shall make it 5m